c/README.md

```diff
@@ -1,5 +1,26 @@
 # libxenserver usage examples
 
+## Overview
+
+The following simple examples are included in this repository:
+
+-  `test_enumerate`: Shows how to enumerate the various API objects.
+
+-  `test_event_handling`: Shows how to listen for events on a connection.
+
+-  `test_get_records`: Shows how to obtain information on API objects such as 
+    hosts, VMs, and storage repositories.
+
+-  `test_failures`: Shows how to translate between error strings and
+    `enum_xen_api_failure`.
+
+-  `test_vm_async_migrate`: Shows how to use asynchronous API calls to migrate 
```

Comment:
nit: language to use coordinator/supporter? I know we're not consistent but we can address this one?